wire-ios-sync-engine/Source/Use cases/SnoozeCertificateEnrollmentUseCase.swift

```diff
@@ -20,7 +20,7 @@ import Foundation
 
 // sourcery: AutoMockable
 public protocol SnoozeCertificateEnrollmentUseCaseProtocol {
-    func invoke() async
+    func invoke(isUpdateMode: Bool) async
```

Comment:
Please add a description for the parameter.
```suggestion
    /// <#Description#>
    /// - Parameter isUpdateMode: <#isUpdateMode description#>
    func invoke(isUpdateMode: Bool) async
```

Reply:
done